static/src/javascripts-legacy/projects/common/modules/article/space-filler.js

```diff
@@ -32,7 +28,7 @@ define([
      */
     SpaceFiller.prototype.fillSpace = function (rules, writer, options) {
         var write = (options && options.domWriter) || fastdom.write;
-        return queue.add(insertNextContent);
+        return this.queue.then(insertNextContent).catch(onError);
```

Comment:
You are right - I didn't think about that the promises should run in the right order. 👍